pkg/maps/ctmap/ctmap.go

```diff
@@ -575,11 +575,17 @@ func (f *GCFilter) doFiltering(srcIP, dstIP netip.Addr, srcPort, dstPort uint16,
 }
 
 func doGC(m *Map, filter *GCFilter) int {
+	if m.mapType.isIPv4() {
+		stats := doGC4(m, filter)
+		m.UpdatePressureMetricWithSize(int32(stats.aliveEntries))
+		return int(stats.deleted)
+	}
 	if m.mapType.isIPv6() {
```

Comment:
IIUC, this code didn't exist before, as in we didn't have GC for IPv4 maps?

Reply:
Tx Chris! No ipv4 gc checks was there.